data/afaanoromoo/bbc_afaanoromoo_radio/schedule.json

```diff
@@ -0,0 +1,1268 @@
+{
```

Comment:
This filename should be `data/afaanoromoo/bbc_oromoo_radio/schedule.json`

Reply:
Should we have two data directories for afaanoromoo?
`data/afaanoromoo/bbc_oromoo_radio/` and `data/afaanoromoo/bbc_afaanoromoo_radio/`